### ISSUE TITLE
Failed with "too many log files open"

### ISSUE THREAD

**Issue by aakepley** [low-sky/degas#9]
See below for error message:
```
Now Processing Scan 140 for Feed 8                                                     
Now Processing Scan 141 for Feed 8                                                     
---------------------------------------------------------------------------            
IOError                                   Traceback (most recent call last)            
<ipython-input-9-83e7e2e97fd6> in <module>()                                           
----> 1 degas.calscans(dataFileName, start=99, stop=145, refscans=[97],outdir=dataOutDir)                                                                                     

/users/akepley/anaconda2/lib/python2.7/site-packages/degas/arguspipe.pyc in calscans(inputdir, start, stop, refscans, outdir)                                                 
    153                         columns = tuple(pipe.infile[ext].get_colnames())       
    154                         integs = gbtpipe.ConvenientIntegration(\               
--> 155                             pipe.infile[ext][columns][rows])                   
    156                         timestamps = integs.data['DATE-OBS']                   
    157                         elevation = np.median(integs.data['ELEVATIO'])         

/users/akepley/anaconda2/lib/python2.7/site-packages/gbtpipe/ConvenientIntegration.pyc in __init__(self, row, log)                                                            
     11         self.weather = Weather.Weather()                                       
     12         if not log:                                                            
---> 13             self.log = Logging()                                               
     14     def mjd(self):                                                             
     15         return self.pu.dateToMjd(self.data['DATE-OBS'][0])                     

/users/akepley/anaconda2/lib/python2.7/site-packages/gbtpipe/PipeLogging.pyc in __init__(self, verbose, prefix, toconsole)                                                    
     41                                                                                
     42         logfilename = prefix + '_' + self.timestamp() + '.log'                 
---> 43         self.configure_logfile(verbose, logfilename, toconsole)
     44
     45     def timestamp(self):

/users/akepley/anaconda2/lib/python2.7/site-packages/gbtpipe/PipeLogging.pyc in configure_logfile(self, verbose, logfilename, toconsole)
    108
    109         # create file handler which logs even debug messages
--> 110         fh = logging.FileHandler(filename='log/'+logfilename, mode='w')
    111         fh.setLevel(logging.DEBUG)
    112         fh_formatter = logging.Formatter("%(asctime)s - %(levelname)s - %(message)s")

/users/akepley/anaconda2/lib/python2.7/logging/__init__.pyc in __init__(self, filename, mode, encoding, delay)
    911             self.stream = None
    912         else:
--> 913             StreamHandler.__init__(self, self._open())
    914
    915     def close(self):

/users/akepley/anaconda2/lib/python2.7/logging/__init__.pyc in _open(self)
    941         """
    942         if self.encoding is None:
--> 943             stream = open(self.baseFilename, self.mode)
    944         else:
    945             stream = codecs.open(self.baseFilename, self.mode, self.encoding)

IOError: [Errno 24] Too many open files: '/home/scratch/akepley/gbt_hcn/degas/log/gbtpipe_14.02.2017_14:20:00.log'
```

**Comment by low-sky** [comment]
This should be addressed in 0dae809 however, I will leave this open for a while and please let me know if the issue repeats.

**Comment by aakepley** [comment]
Ran script which triggered this before and it ran to completion so I'm closing this ticket.
